### ISSUE TITLE
failed to build program

### ISSUE THREAD

**Issue by PNixx** [todxx/teamredminer#3]
Ubuntu, latest AMD driver (amdgpu-pro-18.30-641594):

```
          Team Red Miner version 0.3.2
failed to build program.
[2018-10-01 20:23:26] Failed to initialize device number 0.
```

Command: 
```
./teamredminer --algo=lyra2z --url=stratum+tcp://lyra2z.in.nicehash.com:3365 --user=USER --pass=x --platform=0 -d 0
```

Works correctly with tdxminer.

**Comment by todxx** [comment]
We've heard of other people having issues with the 18.30 driver as well, including with other miners.  Can you post your clinfo output?  That will help us in trying to reproduce this problem.

**Comment by todxx** [comment]
I don't know why AMD needs to change their opencl drivers every damn release.
It looks like starting in 18.30 they moved to a ROCm based driver instead of the PAL driver in 18.20 (which moved to the PAL driver after previous releases had a ROCm based driver for vega...)

The last version we released, v0.3.2, does not support ROCm.  We will have ROCm support again in the next release.  For the time being, for ROCm based drivers, you can use the previous release, v0.3.1, as the only new features in v0.3.2 are windows support and vega PAL driver support.

**Comment by todxx** [comment]
Version v0.3.3 should now support all recent driver versions.  Please let me know if you're still having troubles.
